1.2/sd-client/src/main/java/ServiceDirectoryClient.java

```diff
@@ -34,11 +41,15 @@
                 build(); //boolean option
         Option server = Option.builder("s").longOpt("server").required(false).numberOfArgs(1).desc(
                 "the server address and port").argName("host:port").build();
-        Option command = Option.builder("exec").longOpt("command").required(true).numberOfArgs(1)
+        Option command = Option.builder("exec").longOpt("command").required(false).numberOfArgs(1)
                 .desc("the command and arguments to execute")
                 .build();
+        Option output = Option.builder("o").longOpt("output").required(false).numberOfArgs(1)
+                .desc("execution output fle location. output to STDOUT by default")
```

Comment:
typo. How about saying "Output to the file as specified. The default output is STDOUT."?